.github/workflows/gh-runner-release.yml

```
@@ -0,0 +1,43 @@
+name: Release gh-runner
+
+on:
+  push:
+    tags:
+      - "*-ghrunner"
```

Changing from postfix to prefix groups the tags together
```suggestion
      - "ghrunner-*"
```